layouts/partials/head.html

```diff
@@ -25,4 +47,24 @@
     onload="this.media='all'" />
 
   <link rel="shortcut icon" href="{{ .Site.BaseURL }}favicon.ico" />
+  <meta name="theme-color" content="#e3ddeef2" />
+  <link rel="manifest" href="manifest.json" />
+  <script src="sw.js"></script>
```

Comment:
Sorry for being late to the party, but is this line necessary?

My understanding of service worker is that you load them via `register`, as you've done on line 56 below?

Reply:
I have a bigger issue with this worker actually, which is 

18sw.js:36 Uncaught (in promise) TypeError: Failed to execute 'clone' on 'Response': Response body is already used

I will do a revision one mo